pkg/storaged/client.js

```diff
@@ -1144,6 +1144,7 @@
                 proxy.StoppedPools[uuid] = {
                     devs: l.devs,
                     key_description: { t: "(bv)", v: [true, l.key_description] },
+                    clevis_info: { t: "(bv)", v: [true, l.clevis_info] },
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18818-20230524-165921-2ee4ffa3-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)